.github/workflows/deno.yml

```
@@ -0,0 +1,44 @@
+name: Deno
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

The tests should always be run, not only on the main branch. Deno is rather fast so its not even taking that long and I prefer duplicate runs over missing runs. Other thoughts here?

```suggestion
  push:
  pull_request:
```

---

Sure. Still, I was only re-using the scripts/configs that I've been using before. But it is your repository and I agree with what you said.